src/traverse_path.rs

```diff
@@ -13,7 +13,7 @@ const TRAVERSE_COST_PER_BIT: Cost = 4;
 
 // return a bitmask with a single bit set, for the most significant set bit in
 // the input byte
-fn msb_mask(byte: u8) -> u8 {
+pub fn msb_mask(byte: u8) -> u8 {
```

Comment:
I'm not sure, but I think you can say `pub(crate)` to avoid exporting them. If so, it would make sense for these functions

Reply:
done